CONTRIBUTING.md

```diff
@@ -210,6 +203,14 @@ follow the instructions in this section.
 1. Push the tag: `git push vx.y.z` (or `git push origin vx.y.z` if you are working
    from your local machine).
 
+### Push Helm package
+1. The tagged build will package the Secrets Provider Helm chart for us. The package will appear under the 'Artifacts' tab of the Jenkins build and will resemble `secrets-provider-<version>.tgz`. 
+Save this file you will need it for the next step.
```

Comment:
comma is missing i think.
Save this file **,** you will need it for the next step.

Reply:
great catch! I added a "because"